dlab_core/infrastructure/repositories.py

```diff
@@ -183,12 +295,24 @@ def __init__(self, arg_parse=None):
 
```

Comment:
line 291
maybe it would be better to use ternary operator like?
self.arg_parse = argparse.ArgumentParser() if arg_parse is None else arg_parse

Reply:
self.arg_parse = arg_parse or argparse.ArgumentParser()
will be better